packages/nimble-components/docs/component-status.stories.ts

```diff
@@ -280,64 +285,64 @@ const components = [
         designLabel: 'Figma',
         issueHref: 'https://github.com/ni/nimble/issues/297',
         issueLabel: 'Issue',
-        componentStatus: '✅',
-        angularStatus: '✅',
-        blazorStatus: '✅',
+        componentStatus: ComponentFrameworkStatus.ready,
+        angularStatus: ComponentFrameworkStatus.ready,
+        blazorStatus: ComponentFrameworkStatus.ready,
     },
     {
         componentName: 'Rich Text Editor',
         componentHref: './?path=/docs/incubating-rich-text-editor--docs',
         issueHref: 'https://github.com/ni/nimble/issues/1288',
         issueLabel: 'Issue',
-        componentStatus: '⚠️',
-        angularStatus: '⚠️',
-        blazorStatus: '⚠️',
+        componentStatus: ComponentFrameworkStatus.incubating,
+        angularStatus: ComponentFrameworkStatus.incubating,
+        blazorStatus: ComponentFrameworkStatus.doesNotExist,
```

Comment:
I changed the `blazorStatus` of the Rich Text Editor and Rich Text Viewer to "does not exist" rather than "incubating" since Blazor wrappers haven't been created for those components (not is there an immediate plan to create them).

All other status for other components were left unchanged.